tedana/tests/test_t2smap.py

```diff
@@ -118,6 +116,33 @@ def test_basic_t2smap4(self):
         img = nib.load(op.join(out_dir, 'ts_OC.nii.gz'))
         assert len(img.shape) == 4
 
+    def test_t2smap_cli(self):
+        """
+        Run test_basic_t2smap1, but use the CLI method.
+        """
+        data_dir = get_test_data_path()
+        data = [op.join(data_dir, 'echo1.nii.gz'),
+                op.join(data_dir, 'echo2.nii.gz'),
+                op.join(data_dir, 'echo3.nii.gz')]
+        args = (['-d'] + data +
+                ['-e', '14.5', '38.5', '62.5', '--combmode', 't2s',
+                 '--fitmode', 'all', '--label', 't2smap'])
+        workflows.t2smap._main(args)
+        out_dir = 'TED.echo1.t2smap'
+
+        # Check outputs
+        assert op.isfile(op.join(out_dir, 'ts_OC.nii.gz'))
```

Comment:
Do we need this line since you're loading the file below (L143)? That check should fail if it doesn't exist.

Reply:
Good catch. Will change.